test/mako/config/benchmark.go

```diff
@@ -38,13 +40,38 @@ func MustGetBenchmark() (*string, *string) {
 	return benchmarkKey, benchmarkName
 }
 
+type BenchmarkEnv struct {
+	DevKey   string `envconfig:"DEV_BENCHMARK_KEY"`
+	DevName  string `envconfig:"DEV_BENCHMARK_NAME"`
+	ProdKey  string `envconfig:"PROD_BENCHMARK_KEY"`
+	ProdName string `envconfig:"PROD_BENCHMARK_NAME"`
+}
+
 // getBenchmark fetches the appropriate benchmark_key for this configured environment.
 func getBenchmark() (*string, *string, error) {
 	// Figure out what environment we're running in from the Mako configmap.
 	env, err := getEnvironment()
 	if err != nil {
 		return nil, nil, err
 	}
+	// If pod has benchmark related env
+	benchmarkEnv := BenchmarkEnv{}
+	if err := envconfig.Process("", &benchmarkEnv); err != nil {
+		return nil, nil, err
+	}
+	if !reflect.DeepEqual(benchmarkEnv, BenchmarkEnv{}) {
+		if env == "dev" {
+			if len(benchmarkEnv.DevKey) != 0 && len(benchmarkEnv.DevName) != 0 {
+				return &benchmarkEnv.DevKey, &benchmarkEnv.DevName, nil
+			}
+			return nil, nil, fmt.Errorf(("not enough env in aggregator"))
```

Comment:
Please change the error message to something like `failed to get benchmark config from environment variables`, since this function is used by multiple projects and only `eventing` has `aggregator`.
Also there should only be one pair of brackets :)